packages/device_info_plus/device_info_plus/lib/src/model/linux_device_info.dart

```diff
@@ -139,8 +139,11 @@ class LinuxDeviceInfo implements BaseDeviceInfo {
   /// decoded from hexadecimal, this corresponds to a 16-byte/128-bit value.
   final String? machineId;
 
-  /// Serializes [LinuxDeviceInfo] to a map.
-  @Deprecated('[toMap] method will be discontinued')
+  @override
+  // ignore: deprecated_member_use_from_same_package
+  Map<String, dynamic> get data => toMap();
+
+  @Deprecated('Use [data] getter instead')
```

Comment:
With dart plugins, they can implement the `BaseDeviceInfo` directly but still need to provide the `data`. At the moment, there's no warranty that the `data` is serializable, but that's out of scope for this PR.